benchexec/tablegenerator/template.html

```diff
@@ -1423,6 +1430,11 @@
   showScatterPlot();
 };
 
+function toggleFloatThead() {
+    $(".collapsable-header").toggle();
+    $("#dataTable").floatThead('reflow');
```

Comment:
It is actually problematic if there is a change in the width of table cells, e.g., in the following flow:
- load table
- filter rows with status `true`
- toggle header and trigger reflow
- reset row filter such that cells with status `error (something)` reappear

Because the reflow has shrunk the table cells to the width according to their current content, the last step needs to increase the width of the table cells again, and thus the alignment with the header is lost.